### PR TITLE
doc: Proofreading rdisc.xml

### PR DESCRIPTION
**Pull request by phkam**
A few suggestions to make the manpage more easy to read

Signed-off-by: Philipp Kammerer <kammerer@b1-systems.de>

### CODE REVIEW

File: doc/rdisc.xml [hunk 1]
```diff
@@ -61,22 +61,22 @@ xml:id="man.rdisc">
...
+    consideration if the received
+    <emphasis remap="B">advertise</emphasis> message has a
+    preference of maximum negative.</para>
     <para>Server side of router discovery protocol is supported by
     Cisco IOS and by any more or less complete UNIX routing daemon,
-    f.e
-    <emphasis remap="B">gated</emphasis>. Or,
-    <command>rdisc</command> can act as responder, if compiled with
-    ENABLE_RDISC_SERVER.</para>
+    e.g.
```
Comment (kerolasa) [inline doc/rdisc.xml]: Rather than shorthand of 'exempli gratia' how about 'For example'.

Comment (pevik) [reply doc/rdisc.xml]: I consider *e.g.* as quite common abbreviation, but *for example* obviously ok as well.

**Comment by pevik** [comment]
@phkam can you also fix *the the* in the text?
And add space between *Alexey Kuznetsov* and *&lt;kuznet@ms2.inr.ac.ru&gt;*
The rest looks good to me. Thanks for your patch.

@kerolasa feel free to merge it with:
`Reviewed-by: Petr Vorel <petr.vorel@gmail.com>`

File: doc/rdisc.xml (hunk 1 above)
Comment (phkam) [reply doc/rdisc.xml]: Me, too. So consider this done.

**Comment by kerolasa** [comment]
Thank you for contribution @phkam changes are merged, 7acef2aad2fc95b2838db1180f539fe2b0aa7077

**Pull request closed without merging**